e2e/commands/commit.e2e.js

```diff
@@ -94,6 +94,13 @@ describe('bit tag command', function () {
         const tagWithExisting = () => helper.commitComponent('components/exact', 'message', '-f --exact_version 5.5.5');
         expect(tagWithExisting).to.throw('the version 5.5.5 already exists for components/exact');
       });
+      it('Should print same output for flaged tag and non flaged tag', () => {
+        const majorOutput = helper.commitComponent('components/major', 'message', '--major');
```

Comment:
don't use the components/major because it's already committed.
Create a new one.